src/components/Badge.tsx

```diff
@@ -28,12 +28,16 @@ interface BadgeProps {
   className?: string
   amount?: string | bigint | undefined
   prefix?: string
+  inline?: boolean
   floatRight?: boolean
   minWidth?: number
 }
 
-let Badge = ({ content, className, amount, prefix, minWidth, floatRight = false }: BadgeProps) => (
-  <div className={className} style={{ float: floatRight ? 'right' : 'left', minWidth }}>
+let Badge = ({ content, className, amount, prefix, minWidth, inline = false, floatRight = false }: BadgeProps) => (
+  <div
+    className={className}
+    style={{ display: inline ? 'inline' : 'block', float: inline ? 'none' : floatRight ? 'right' : 'left', minWidth }}
```

Comment:
This is getting quite long and ugly, how about moving it to the styled component definition below? We already use the `type` prop, we can also use the `inline` and `floatRight` there as well.

Reply:
Done!